# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundParentImpl.h
# Commit: 7a4e3f5f674a
# Full Hash: 7a4e3f5f674ad675fa084bf1f30a79a15e8c7d2a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - PFetch protocol declaration and implementation. r=dom-worker-reviewers,jesup
#   
#   Depends on D138813
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142436
# ==============================================================================

diff -r 9da00c1364a5 -r 7a4e3f5f674a ipc/glue/BackgroundParentImpl.h
--- a/ipc/glue/BackgroundParentImpl.h	Wed Jan 18 00:50:19 2023 +0000
+++ b/ipc/glue/BackgroundParentImpl.h	Wed Jan 18 00:50:19 2023 +0000
@@ -415,6 +415,8 @@
 
   already_AddRefed<PLockManagerParent> AllocPLockManagerParent(
       const ContentPrincipalInfo& aPrincipalInfo, const nsID& aClientId) final;
+
+  already_AddRefed<PFetchParent> AllocPFetchParent() override;
 };
 
 }  // namespace mozilla::ipc